src/SFA.DAS.FAA.Web/Views/SearchApprenticeships/SearchResults.cshtml

```diff
@@ -164,7 +164,7 @@
                                 New
                             </strong>
                         }
-
+                        @Html.Raw(vacancy.VacancyStatusBadge)

```

Comment:
From a security point of view this will most likely come up - can we look at refactoring it into a tag helper or to just have the condition in view?

Reply:
Resolved it with TagHelpers